dts/arm/silabs/efr32bg22-pinctrl.dtsi

```diff
@@ -10,10 +10,9 @@
 	/* configuration for uart0 device, default state */
 	usart0_default: usart0_default {
 		group1 {
-			/* configure PE.11 as UART_RX and PE.10 as UART_TX */
-			psels = <GECKO_PSEL(UART_TX, E, 10)>,
-				<GECKO_PSEL(UART_RX, E, 11)>,
-				<GECKO_LOC(UART, 0)>;
+			psels = <GECKO_PSEL(SPI_SCK, C, 2)>,
+				<GECKO_PSEL(SPI_MOSI, C, 0)>,
+				<GECKO_PSEL(SPI_MISO, C, 1)>;
```

Comment:
non-blocking: maybe a comment saying UART is operating as SPI here.

Reply:
Added comments